{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/graphql/user_schema.py

```diff
@@ -0,0 +1,30 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required, superuser_required
+
+from {{cookiecutter.main_module}}.users.models import User
+
+
+class UserType(DjangoObjectType):
+    class Meta:
+        model = User
+        fields = ["id", "first_name", "last_name", "email"]
+
+
+class UserQueries(graphene.ObjectType):
+    me = graphene.Field(
+        UserType,
+        description='Return Current User\'s Information'
+    )
+    users = graphene.List(
+        UserType,
+        description='Return List of all Users'
+    )
```

Comment:
We can just use double-quotes here instead of escaping single quotes because of enclosing single-quotes.

Reply:
Resolved in https://github.com/Fueled/django-init/pull/452/commits/40c0c49fc1192cf3526fda83a61121bc4b4cc2d5